### DISCUSSION TITLE
[Question][Hw1] Question about line ending

### DISCUSSION THREAD

**Discussion by mindol** [prosyslab-classroom/cs424-program-reasoning#290] [Q&A]
Name: Minsol Park(박민솔)

I think I have a problem about line endings, and I want to get some tips about this problem.

![image](https://github.com/prosyslab-classroom/cs492-program-reasoning/assets/14192966/14b1c52c-9437-4241-9fec-e25784ad7701)
I use 'diff' to compare example1.output and example1.expected and this is the result.

It seems same integer '39', but I think the line ending is a problem, because of the above warning in the picture.
Also, it seems .expected file is CRLF ending, and .output file is LF ending(checked by VS Code)

How can I make my interpreter make CRLF ending outputs?
Or can I get some tips about this problem?

**Comment by doit-man** [comment]
For me, interpreter's output is end with LF(0xa). (in docker. just plain print) 
```
$ xxd example1.output
00000000: 3339 0a                                  39.
```
and also, example1.expected is end with LF(0xa).
```
$ xxd example1.expected
00000000: 3339 0a                                  39.
```

**Comment by mindol** [comment]
@doit-man I just fixed my .expected files to have LF ending, then everything works well!
I think something went wrong during my coding, so .expected files accidentally became CRLF ending.
Thank you for reply!
